configs/detection/lfb/i3d_for_lfb.py

```diff
@@ -0,0 +1,142 @@
+# model settings
```

Comment:
What is this config for? If it is used as the pre-train weight for lfb detector, you need to tell the performance difference on Kinetics-400 and AVA between (1, 2, 2, 2) i3d and (1, 2, 2, 1) i3d.